app/frontend/styles/chakra-overrides.scss

```diff
@@ -6,27 +6,26 @@
 
```

Comment:
as per @taw27 we should re-evaluate this file and move to the theme if possible so overrides work as expected.